src/components/App.tsx

```diff
@@ -6141,6 +6166,29 @@ class App extends React.Component<AppProps, AppState> {
   };
 
   private getContextMenuItems = (
+    type: "canvas" | "element" | "custom",
+    source?: string,
+  ): ContextMenuItems => {
+    const custom: ContextMenuItems = [];
+    this.actionManager
+      .getCustomActions({ data: { source: source ?? "" } })
+      .forEach((action) => custom.push(action));
+    if (type === "custom") {
+      return custom;
+    }
+    if (custom.length > 0) {
+      custom.push(CONTEXT_MENU_SEPARATOR);
+    }
```

Comment:
rendering custom contextMenu will have to be done differently https://github.com/excalidraw/excalidraw/issues/6139

Reply:
I have removed this.